bundles/framework/printout/PrintoutHandler.js

```diff
@@ -18,7 +18,10 @@ class UIHandler extends StateHandler {
             scaleType: 'map',
             scale: null,
             showTimeSeriesDate: false,
-            isMapStateChanged: false
+            isMapStateChanged: false,
+            showCoordinates: false,
```

Comment:
```suggestion
            // impl for toggling coordinates is commented out in JSX. This will be enabled on future release.
            showCoordinates: false,
```